pallets/balances/src/lib.rs

```diff
@@ -335,10 +338,17 @@ decl_module! {
             Self::safe_transfer_core(&transactor, &dest, value, None, ExistenceRequirement::AllowDeath)?;
         }
 
+        // TODO: X Read of the cdd check for destination account.
```

Comment:
It is related to the weight calculation. As the upstream module has the weight mgt, because of our changes, we are messing up with those calculation so I added a TODO as a remainder to fix when we do a proper benchmarking.

Reply:
Can you please update the TODO to say something like that? It's a bit confusing right now.